audio_service/lib/audio_service.dart

```diff
@@ -3045,17 +3095,16 @@ class AudioServiceBackground {
   /// Sends a custom event to the Flutter UI.
   ///
   /// The event parameter can contain any data permitted by Dart's
-  /// SendPort/ReceivePort API. Please consult the relevant documentation for
+  /// [SendPort]/[ReceivePort] API. Please consult the relevant documentation for
```

Comment:
This slows down dartdoc significantly, and in the past I have also seen it error out. A couple of months ago, this was also causing a serious problem on pub.dev causing the publish command to fail for many plugin developers.

Not sure on the current state of things...

Reply:
(to clarify, referencing docs in other packages)

Reply:
this should work, if it didn't this was some temporary error in pub.dev. definitely not a reason to not use that

Reply:
Waiting 10 minutes to test documentation changes was really disrupting my productivity, but let's see. Run dartdoc just to check that the issue has been rectified.

Reply:
i ran dartdoc - no problem with this
i also patched some unresolved references